docs/specs/metadata.yml

```diff
@@ -57,6 +57,26 @@ outputs:
     { "redirectionUrl":"/sccm/mdm/understand/hybrid-mobile-device-management" }
   build.manifest:
 ---
+# generate same document id and version id with 'redirection with document id' evem it's out of build scope
```

Comment:
how about merging the other `redirection-out-of-scope` test in here? feels like this covers the other test